### ISSUE TITLE
SASComputeDeleteSession session_name parameter name

### ISSUE THREAD

**Issue by nicrobert** [sassoftware/sas-airflow-provider#27]
It looks like the parameter to specify the session name is "session_name" in the class header. However, it does not work and seems to be "compute_session_name".

To be aligned with SASComputeCreateSession, "session_name" should be preferable.

**Comment by AndrewShakinovsky-SAS** [comment]
I don't think that the session_name in SASComputeCreateSession and the session id (I assume you meant compute_session_id because I don't see a compute_session_name) are referring to the same thing.
In SASComputeCreateSession the session_name is the name you want to give the session whereas the compute_session_id is the actual id that was assigned to it by compute. The studio operator does not (currently) allow you to assign a session name, it only allows passing in of a pre-existing id, or to try to look up the session id using the default "Airflow-Session" name.

**Comment by nicrobert** [comment]
"compute_session_name" in sas_delete_session.py line 42 is mismatched with "session_name" as documented in line 34 of the same file.
